packages/supabase/.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+# Keep environment variables out of version control
+.env
```

Comment:
this will only add a file that exactly matchs `.env`. I think it's a good practice to also add `.env*` in case anyone adds a file like `.env.development` or `.env.production`. but it's up to you since we don't have those files yet.

Reply:
I agree with @eulixir, our .env files are for development only